main.go

```diff
@@ -30,11 +30,19 @@ import (
 	"github.com/google/uuid"
 )
 
-var xdsServerUri = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
-var outputName = flag.String("output", "-", "output file name")
-var gcpProjectNumber = flag.Int64("gcp-project-number", 0,
-	"the gcp project number. If unknown, can be found via 'gcloud projects list'")
-var vpcNetworkName = flag.String("vpc-network-name", "default", "VPC network name")
+var (
+	xdsServerUri     = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
+	outputName       = flag.String("output", "-", "output file name")
+	gcpProjectNumber = flag.Int64("gcp-project-number", 0,
+		"the gcp project number. If unknown, can be found via 'gcloud projects list'")
+	vpcNetworkName = flag.String("vpc-network-name", "default", "VPC network name")
+
+	// Notice: This flag is EXPERIMENTAL and may be changed or removed in a later release.
+	enableFileWatcherConfig = flag.Bool("enable-file-watcher-config", false, "whether or not to generate file_watcher certificate provider config")
+)
+
+// For overriding in unit tests.
+var newUUIDString = func() string { return uuid.New().String() }
```

Comment:
Could we use the fixed seed instead? That way, we actually see that, yes, it is a UUID.

Reply:
Done.